pkg/aws/resources_container.go

```diff
@@ -38,6 +39,7 @@ func NewResourcesContainer() *ResourcesContainer {
 		SecurityGroupsList: []*aws2.SecurityGroup{},
 		SubnetsList:        []*aws2.Subnet{},
 		VpcsList:           []*aws2.Vpc{},
+		Provider:           "aws",
```

Comment:
How about moving the `AWS` and `IBM` consts from `pkg/factory` to `pkg/common` and use them here as well?